packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2441,15 +2441,16 @@ class _RenderInspectorOverlay extends RenderBox {
     context.addLayer(_InspectorOverlayLayer(
       overlayRect: Rect.fromLTWH(offset.dx, offset.dy, size.width, size.height),
       selection: selection,
+      rootRenderObject: parent as RenderObject,
```

Comment:
to be defensive, maybe add an is check in case the parent is not a RenderObject.